Ombi.Api/TheMovieDbApi.cs

```diff
@@ -100,5 +105,48 @@ public TmdbMovieDetails GetMovieInformationWithVideos(int tmdbId)
             var movies = await Client.GetMovie(imdbId);
             return movies ?? new Movie();
         }
+
+        public async Task<List<Movie>> SearchActor(string searchTerm)
+        {
+            return await SearchActor(searchTerm, null);
+        }
+
+        public async Task<List<Movie>> SearchActor(string searchTerm, Func<int, string, string, Task<bool>> alreadyAvailable)
+        {
+            SearchContainer<SearchPerson> result = await Client.SearchPerson(searchTerm);
+            
+            var people = result?.Results ?? new List<SearchPerson>();
+            var person = (people.Count != 0 ? people[0] : null);
```

Comment:
This fixes the issue with the lack of a "no results" message.  The old way I was doing this would try getting element 0 of an empty result set and the API call was 500'ing to the client... so yeah.